flow/project.py

```diff
@@ -3512,22 +3604,11 @@ def _verify_group_compatibility(cls, groups):
         """Verify that all selected groups can be submitted together."""
         return all(a.isdisjoint(b) for a in groups for b in groups if a != b)
 
-    def _aggregate_is_in_project(self, aggregate):
-        """Verify that the aggregate belongs to this project."""
-        return any(
-            get_aggregate_id(aggregate) in aggregates
-            for aggregates in self._group_to_aggregator.inverse
-        )
-
-    @staticmethod
-    def _is_selected_aggregate(aggregate, jobs):
-        """Verify whether the aggregate is present in the provided jobs."""
-        return aggregate in jobs
-
     def _get_aggregate_from_id(self, id):
         # Iterate over all the instances of stored aggregates and search for the
         # aggregate id in those instances.
-        for aggregate_store in self._group_to_aggregator.inverse:
+        # aggregate in those instances.
```

Comment:
These two comments are confusing.

Reply:
Haha, good catch. I made a mistake in resolving merge conflicts. Fixed.